client/src/main/java/org/evosuite/ga/Chromosome.java

```diff
@@ -537,4 +546,28 @@ public double getCoverageInstanceOf(Class<?> clazz) {
 		}
 		return 0.0;
 	}
+
```

Comment:
Can you maybe just denote the methods used only for CRO as such? (Comment or something like that)

Reply:
Done.